ServerCore/Pages/Events/Players.cshtml

```diff
@@ -0,0 +1,61 @@
+﻿@page "/{eventId}/Events/Players"
+@model ServerCore.Pages.Events.PlayersModel
+@{
+    ViewData["Title"] = "Players";
+}
+
+<h2>Players</h2>
+
+<table class="table">
+    <thead>
+        <tr>
+            <th>
+                Player @Html.DisplayNameFor(model => model.Players[0].Member.ID)
```

Comment:
Do we need to see ids on this page?

Reply:
I'm going to leave these in to help with development / debugging - I'll take them back out once we get to pretty-ifying the UI or user testing. Opened issue 99 to track this.